modules/kubernetes-addons/prometheus/locals.tf

```diff
@@ -72,10 +47,50 @@ locals {
     operating_system = "linux",
   })]
 
+  amazon_prometheus_workspace_id           = var.amazon_prometheus_workspace_id == null && var.enable_amp_for_prometheus ? aws_prometheus_workspace.amp_workspace[0].id : var.amazon_prometheus_workspace_id
+  amp_workspace_url                        = var.enable_amp_for_prometheus && local.amazon_prometheus_workspace_id != null ? "https://aps-workspaces.${data.aws_region.current.id}.amazonaws.com/workspaces/${local.amazon_prometheus_workspace_id}/api/v1/remote_write" : null
+  amazon_prometheus_ingest_iam_role_arn    = var.enable_amp_for_prometheus ? module.irsa.*.ingest.irsa_iam_role_arn[0] : null
+  amazon_prometheus_ingest_service_account = local.irsa_config.ingest.service_account
```

Comment:
naming consistency with these variables. Either go with `amp` as a prefix or `amazon_prometheus`. I am partial to `amp` for brevity.

Reply:
Sure, I have told by the teams not to use `amp` . I will change this to `amazon_prometheus`